src/js/angular/clustermanagement/controllers/edit-cluster-nodes-modal.controller.js

```diff
@@ -23,9 +24,25 @@ function EditClusterNodesModalController($scope, $uibModalInstance, $translate,
 
     /**
      * Confirms and saves the changes, closing the modal.
+     *
+     * This function is called when the user confirms their actions in the modal.
+     *
+     * It retrieves the current cluster view from `ClusterContextService`.
+     *
+     * If the cluster does not have a valid cluster configuration,
+     * it closes the modal and passes a deep clone of the updated cluster configuration.
+     *
+     * If the cluster does have a valid configuration, it passes a deep clone of the full update actions.
+     *
+     * @private
```

Comment:
just to mention here that this is not @private

Reply:
will fix it in the next MR